<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-11-19T11:59:19 IST" name="Suite" finished-at="2024-11-19T12:01:19 IST" duration-ms="120458">
    <groups>
    </groups>
    <test started-at="2024-11-19T11:59:19 IST" name="TestNg" finished-at="2024-11-19T12:01:19 IST" duration-ms="120458">
      <class name="testcases.TC_MC_001">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.TC_MC_001@5e5d171f]" started-at="2024-11-19T12:00:03 IST" name="setup" finished-at="2024-11-19T12:00:09 IST" duration-ms="5922" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="TC_MC_001testCase()[pri:0, instance:testcases.TC_MC_001@5e5d171f]" started-at="2024-11-19T12:00:09 IST" name="TC_MC_001testCase" finished-at="2024-11-19T12:00:14 IST" duration-ms="5185" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Case Page Message Did Not Match expected [My Casess] but found [My Cases]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Case Page Message Did Not Match expected [My Casess] but found [My Cases]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testcases.TC_MC_001.TC_MC_001testCase(TC_MC_001.java:45)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_MC_001testCase -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.TC_MC_001@5e5d171f]" started-at="2024-11-19T12:00:14 IST" name="teardown" finished-at="2024-11-19T12:00:15 IST" duration-ms="1028" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.TC_MC_001 -->
      <class name="testcases.TC_MC_004">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.TC_MC_004@4c178a76]" started-at="2024-11-19T11:59:19 IST" name="setup" finished-at="2024-11-19T11:59:29 IST" duration-ms="10058" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="TC_MC_004_testCase()[pri:0, instance:testcases.TC_MC_004@4c178a76]" started-at="2024-11-19T11:59:29 IST" name="TC_MC_004_testCase" finished-at="2024-11-19T11:59:45 IST" duration-ms="16334" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[element click intercepted: Element <span class="plusminus2" id="pm" title="Expand"></span> is not clickable at point (24, 124). Other element would receive the click: <div class="popupProcessing">...</div>
  (Session info: chrome=131.0.6778.70)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '23'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c3b54511b40730ed6b883b4e7d12bb71, clickElement {id=f.E7799139F6FD23F6034D578BD01FB783.d.A0D1F7B51FEB0E14949B491EF4504F14.e.276}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.70, chrome: {chromedriverVersion: 131.0.6778.69 (77e2244bbcda..., userDataDir: C:\Users\SIVAKU~1.VEE\AppDa...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57401}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57401/devtoo..., se:cdpVersion: 131.0.6778.70, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (c3b54511b40730ed6b883b4e7d12bb71)] -> xpath: //span[@id='pm']]
Session ID: c3b54511b40730ed6b883b4e7d12bb71]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: element click intercepted: Element <span class="plusminus2" id="pm" title="Expand"></span> is not clickable at point (24, 124). Other element would receive the click: <div class="popupProcessing">...</div>
  (Session info: chrome=131.0.6778.70)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '23'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c3b54511b40730ed6b883b4e7d12bb71, clickElement {id=f.E7799139F6FD23F6034D578BD01FB783.d.A0D1F7B51FEB0E14949B491EF4504F14.e.276}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.70, chrome: {chromedriverVersion: 131.0.6778.69 (77e2244bbcda..., userDataDir: C:\Users\SIVAKU~1.VEE\AppDa...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57401}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57401/devtoo..., se:cdpVersion: 131.0.6778.70, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (c3b54511b40730ed6b883b4e7d12bb71)] -> xpath: //span[@id='pm']]
Session ID: c3b54511b40730ed6b883b4e7d12bb71
at org.testng.Assert.fail(Assert.java:111)
at testcases.TC_MC_004.TC_MC_004_testCase(TC_MC_004.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_MC_004_testCase -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.TC_MC_004@4c178a76]" started-at="2024-11-19T11:59:45 IST" name="teardown" finished-at="2024-11-19T11:59:49 IST" duration-ms="3320" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.TC_MC_004 -->
      <class name="testcases.TC_MC_005">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.TC_MC_005@7770f470]" started-at="2024-11-19T11:59:49 IST" name="setup" finished-at="2024-11-19T11:59:55 IST" duration-ms="6057" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="TC_MC_005_testCase()[pri:0, instance:testcases.TC_MC_005@7770f470]" started-at="2024-11-19T11:59:55 IST" name="TC_MC_005_testCase" finished-at="2024-11-19T12:00:02 IST" duration-ms="6896" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[element click intercepted: Element <span class="plusminus2" id="pm" title="Expand"></span> is not clickable at point (24, 124). Other element would receive the click: <div class="popupProcessing">...</div>
  (Session info: chrome=131.0.6778.70)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '23'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [f259bfd550d765639eb782d2c43157a7, clickElement {id=f.93AE4AF252C9811ED0C1070D750E5771.d.8760E50C443FA812EE859FE0D6E4DB41.e.267}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.70, chrome: {chromedriverVersion: 131.0.6778.69 (77e2244bbcda..., userDataDir: C:\Users\SIVAKU~1.VEE\AppDa...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57502}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57502/devtoo..., se:cdpVersion: 131.0.6778.70, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (f259bfd550d765639eb782d2c43157a7)] -> xpath: //span[@id='pm']]
Session ID: f259bfd550d765639eb782d2c43157a7]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: element click intercepted: Element <span class="plusminus2" id="pm" title="Expand"></span> is not clickable at point (24, 124). Other element would receive the click: <div class="popupProcessing">...</div>
  (Session info: chrome=131.0.6778.70)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '23'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [f259bfd550d765639eb782d2c43157a7, clickElement {id=f.93AE4AF252C9811ED0C1070D750E5771.d.8760E50C443FA812EE859FE0D6E4DB41.e.267}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.70, chrome: {chromedriverVersion: 131.0.6778.69 (77e2244bbcda..., userDataDir: C:\Users\SIVAKU~1.VEE\AppDa...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57502}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57502/devtoo..., se:cdpVersion: 131.0.6778.70, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (f259bfd550d765639eb782d2c43157a7)] -> xpath: //span[@id='pm']]
Session ID: f259bfd550d765639eb782d2c43157a7
at org.testng.Assert.fail(Assert.java:111)
at testcases.TC_MC_005.TC_MC_005_testCase(TC_MC_005.java:56)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_MC_005_testCase -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.TC_MC_005@7770f470]" started-at="2024-11-19T12:00:02 IST" name="teardown" finished-at="2024-11-19T12:00:03 IST" duration-ms="1038" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.TC_MC_005 -->
      <class name="testcases.TC_MC_002">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.TC_MC_002@24313fcc]" started-at="2024-11-19T12:00:15 IST" name="setup" finished-at="2024-11-19T12:00:20 IST" duration-ms="5191" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="TC_MC_002_testCase()[pri:0, instance:testcases.TC_MC_002@24313fcc]" started-at="2024-11-19T12:00:20 IST" name="TC_MC_002_testCase" finished-at="2024-11-19T12:01:18 IST" duration-ms="57696" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_MC_002_testCase -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.TC_MC_002@24313fcc]" started-at="2024-11-19T12:01:18 IST" name="teardown" finished-at="2024-11-19T12:01:19 IST" duration-ms="1324" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.TC_MC_002 -->
    </test> <!-- TestNg -->
  </suite> <!-- Suite -->
</testng-results>
